includes/Data/Patterns.php

```diff
@@ -271,18 +253,14 @@ public static function get_theme_step_patterns_from_step( $step, $squash = false
 		$block_patterns          = array();
 		$block_patterns_squashed = '';
 
-		// fetch the selected header menu slug from DB
-		$flow_data        = \get_option( Options::get_option_name( 'flow' ) );
-		$header_menu_slug = explode( '/', $flow_data['data']['partHeader'] )[1];
-		if ( ! empty( $header_menu_slug ) ) {
-			$pattern_slugs = self::replace_header_menu_slug( $pattern_slugs, $header_menu_slug );
-		}
-
```

Comment:
Any reason for removing this code? cause now the selected header is not appearing on this step "wp-setup/step/design/theme-styles/preview"